eZ/Bundle/EzPublishIOBundle/BinaryStreamResponse.php

```diff
@@ -133,6 +137,11 @@ public function setContentDisposition( $disposition, $filename = '', $filenameFa
      */
     public function prepare( Request $request )
     {
+        if ( !$this->file instanceof BinaryFile )
```

Comment:
Not sure that one is useful any longer.

A `BinaryFile` is required (typehinting) by the ctor as well as `setFile()`. I don't think this situation can happen aymore.n